### DISCUSSION TITLE
Medic Equipment/Customization

### DISCUSSION THREAD

**Discussion by Doobh** [FakeFishGames/Barotrauma#12181] [Ideas and suggestions]
So I’m playing medic in our current campaign and my ONE big complaint is that I started with the last outfit I’d ever wear. It’s a bit disappointing, because those three pocket slots are a big deal!

Considering my head slot is pretty much dedicated to the medic goggles, it generally means I’m not gonna be switching anything out unless I go for the advanced syringe gun (which is locked behind a specific skill tree/faction rep).

I’m not sure I need to make any particular solution ideas because I ain’t a game developer. But if this ends up with more medical equipment variety in general, with pros/cons, that would be pretty awesome. I’d love feeling like I had some more medic focused choices.

**Comment by Ek-Videogames** [comment]
More 'bespoke' clothing variations that play into the specific class roles a bit more would be good for all jobs tbh, not exclusively medic.
